docs/physics/tardisgamma/packetsampling.rst

```diff
@@ -0,0 +1,13 @@
+Packet Sampling
+===============
+
+
+In the simulation, we use spherical symmetry where the ejecta is divided into multiple spherical shells. To find the mass of each shell we multiply the mass fractions of each of the elements and isotopes with the density. We can set the timespace for the simulation
```

Comment:
Is it worth mentioning spherical symmetry? I don't think spherical symmetry is specific to the gamma-ray part of the code. Perhaps just add a hyperlink to https://tardis-sn.github.io/tardis/physics/setup/model.html#Shell-Structure if you want to mention it.

Reply:
I thought it was relevant to mention at least how the ejecta is broken down into shells because it goes into how we find the mass of each shell. I added the hyperlink but if it still doesn't make sense to mention it, I can remove that part.